.github/workflows/core-flow-validation-report.yml

```
@@ -0,0 +1,101 @@
+name: Core Flow Validation Report
+
+on:
+  schedule:
+    - cron: '20 3 * * 2-7'
```

actionlint (1.7.11)

[error] invalid CRON format "20 3 * * 2-7" in schedule event: end of range (7) above maximum (6): 2-7

(events)